linux/docker_virtualenv_ius_apache24.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -e -u -x
+
+#OMEROVER=omero
+OMEROVER=omerodev
+
+source settings.env
+
+#install httpd 2.4
+yum -y install httpd24u
+
+#Create virtual env.
+virtualenv /tmp/omeroenv
```

Comment:
That's what I had before!! then I moved it to /tmp
I will update that. I also need to test https://github.com/openmicroscopy/openmicroscopy/pull/4381#issuecomment-162437684